metrix/src/models.rs

```diff
@@ -1,6 +1,20 @@
 use super::schema::metrics;
 use serde_json;
 use chrono::naive::NaiveDateTime;
+use diesel::sql_types::*;
+
+#[derive(Serialize, Deserialize, Queryable, QueryableByName)]
+pub struct Bucket {
+	#[sql_type = "BigInt"]
+	pub result: i64,
```

Comment:
what's with the uneven indentation, you scrub?

Reply:
I know not of what you speak